### PR TITLE
Raw3-1 Configuration Update

### PR DESCRIPTION
**Pull request by bbrito**
The robot now works with this config.

### CODE REVIEW

File: cob_default_robot_config/raw3-1/arm_joint_configurations.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+joint_names: ['arm_shoulder_pan_joint', 'arm_shoulder_lift_joint', 'arm_elbow_joint','arm_wrist_1_joint', 'arm_wrist_2_joint', 'arm_wrist_3_joint']
```
Comment (fmessmer) [inline cob_default_robot_config/raw3-1/arm_joint_configurations.yaml]: this file needs to be at `cob_default_robot_config/robots/raw3-1/script_server/arm_joint_configurations.yaml`
there is one available already at this location which seems to be identical: https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_default_robot_config/robots/raw3-1/script_server/arm_joint_configurations.yaml

thus, please remove this file from the PR

**Review by fmessmer — CHANGES_REQUESTED** [review]
@ipa-rmb @ipa-bfb 
once the `arm_joint_configurations.yaml` file is removed, I will squash-merge...

File: cob_bringup/robots/raw3-1.xml [hunk 2]
```diff
@@ -40,6 +40,11 @@
 			<include file="$(find cob_bringup)/drivers/emstate_phidget.launch">
 				<arg name="robot" value="$(arg robot)"/>
 			</include>
+			<!-- Relayboard with sim:=True is necessary because the robot does not have a phidgets board for checking the emergency status -->
+			<include file="$(find cob_bringup)/drivers/relayboard.launch">
+				<arg name="robot" value="$(arg robot)"/>
+				<arg name="sim" value="True"/>
+			</include>
```
Comment (fmessmer) [inline cob_bringup/robots/raw3-1.xml]: well, in that case the `X_phidget.launch` files should be removed from the `raw3-1.xml`, as they are not used!
Please also remove the obsolete `X_phidget.yaml` files in `cob_hardware_config`.

---
nit-picking: `True` -> `true`
```
<include file="$(find cob_bringup)/drivers/relayboard.launch">
	<arg name="robot" value="$(arg robot)"/>
	<arg name="sim" value="true"/>
</include>
```

**Review by fmessmer — CHANGES_REQUESTED** [review]

**Comment by fmessmer** [comment]
@ipa-bfb @ipa-rmb
are you able to update/finalize this PR?
(feel free to fetch me in order to finalize together...especially regarding cleaning up the phidgets...)

**Comment by bbrito** [comment]
As soon as we have the new dome running @ipa-rmb will finish this PR ;)

**Comment by ipa-rmb** [comment]
Are you happy with these adaptations or did I miss something?

File: cob_bringup/robots/raw3-1.xml [hunk 3]
```diff
@@ -31,19 +31,19 @@
...
 				<arg name="robot" value="$(arg robot)"/>
 			</include>
 			<include file="$(find cob_bringup)/drivers/powerstate_phidget.launch">
 				<arg name="robot" value="$(arg robot)"/>
 			</include>
 			<include file="$(find cob_bringup)/drivers/emstate_phidget.launch">
 				<arg name="robot" value="$(arg robot)"/>
-			</include>
-			<!-- Relayboard with sim:=True is necessary because the robot does not have a phidgets board for checking the emergency status -->
+			</include-->
```
Comment (fmessmer) [inline cob_bringup/robots/raw3-1.xml]: So, then just to re-confirm: 
 - `raw3-1` indeed does not have **any** phidget-sensor onboard
 - and the robot still works for you with these lines commented out?

If both questions are answered `yes`, then you could also remove the lines completely :wink:

**Review by fmessmer — CHANGES_REQUESTED** [review]
maybe I can have a quick look when you guys are running the robot with this feature branch in order to see whether there is any further issues during start up and basic "robot usage"
Let me know when you are working on it
@ipa-rmb @ipa-bfb

**Comment by ipa-rmb** [comment]
we tested with the robot - it works well

**Review by fmessmer — APPROVED** [review]
finalized together with @ipa-rmb

**Pull request merged**